hepcrawl/extractors/jats.py

```diff
@@ -106,7 +106,13 @@ def _get_authors(self, node):
             for aff in affiliations:
                 # checking is the aff. value captured by xpath is just new line
                 if self._clean_aff(aff).split():
-                    affiliations_values.append({'value': self._clean_aff(aff)})
+                    remove_doubled_commas = "".join(self._clean_aff(aff).split(" ,"))
+                    try:
+                        remove_doubled_commas = re.search('^(,\s)(.*)(,$)', remove_doubled_commas).group(2).strip()
```

Comment:
@ErnestaP  it's weird, this regex doesn't work for me, could we please make a unit test?

Reply:
I should name the regex pattern to reflect what it does: 
'^(,\s)(.*)(,$)' pattern removes the commas in the end and beginning of the string. 
I had to keep the name of`remove_doubled_commas` to have the variable accessible after try/except block. I will improve this part of the code

Reply:
I removed this part completely